project/src/components/place-card-list/place-card-list.tsx

```diff
@@ -0,0 +1,28 @@
+import { useState } from 'react';
+import PlaceCard from '../../components/place-card/place-card';
+
+import { Offer } from '../../types/offer/offer';
+
+type PlaceCardListProps = {
+  offers: Offer[];
+}
+
+export default function PlaceCardList({offers}: PlaceCardListProps): JSX.Element {
+  const [active, setActive] = useState<null | number>(null);
```

Comment:
Если первый параметр не нужен временно, можно опустить
```suggestion
  const [, setActive] = useState<null | number>(null);
```

Reply:
::thumbsup